_docs/extensibility/custom-request-matching.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: docs
-title: Custom Matching
+title: Custom Request Matching
```

Comment:
This all looks fine.  @oleg-nenashev is correct about the redirect though.  Changing the name of the file will change the url structure so all the indexed pages in the search engines will return a 404.  If you take a look at the other pages on the site, some of them have a `redirect_from` element in the front matter.  This is what we need here